docs/src/main/tut/docs/akkahttp.md

```diff
@@ -3,3 +3,102 @@ layout: docs
 title: Akka - Http
 permalink: /docs/integrations/akkahttp/
 ---
+
+#### Akka Http
+
+[Akka HTTP](http://doc.akka.io/docs/akka-http/10.0.5/java/http/introduction.html) (formerly known as `spray`) is an Akka-based library for implementing HTTP services.
```

Comment:
We need a small snippet that shows the sbt dependency to show how to depend in the module. Not needed for this PR but we need to ensure the same is in all docs for all modules whose content comes from one of the `freestyle-*` modules.